os/arch/arm/src/stm32l4/stm32l4_serial.c

```diff
@@ -769,7 +770,7 @@ static struct
   bool serial_suspended;
 } g_serialpm =
   {
-    .pm_cb.name = "serial",
+    .pm_cb.name = SERIAL_NAME,
```

Comment:
This new definition is used only here. Do we need to use this definition only for here?